plugins/aja/aja-common.cpp

```diff
@@ -242,12 +242,13 @@ bool aja_video_format_changed(obs_properties_t *props, obs_property_t *list,
 {
 	UNUSED_PARAMETER(list);
 
-	auto vid_fmt = static_cast<NTV2VideoFormat>(
-		obs_data_get_int(settings, kUIPropVideoFormatSelect.id));
+	const long long vid_fmt =
+		obs_data_get_int(settings, kUIPropVideoFormatSelect.id);
 	size_t itemCount = obs_property_list_item_count(list);
 	bool itemFound = false;
 	for (size_t i = 0; i < itemCount; i++) {
-		int itemFormat = obs_property_list_item_int(list, i);
+		const long long itemFormat =
```

Comment:
What if I cast `itemFormat` to NTV2VideoFormat, and keep the `vid_fmt` cast to NTV2VideoFormat on line 245, in order to make it clear that this property comparison is really an NTV2VideoFormat under the hood?

Reply:
Updated.

Reply:
@paulh-aja Is this acceptable now?